deploy/crds/managed_v1alpha1_subjectpermission_2_cr.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: managed.openshift.io/v1alpha1
```

Comment:
Do we need to commit these? @sam-nguyen7 (There 3 other files are "_3_cr.yaml", "_4_cr.yaml" and  "_5_cr.yaml")

Reply:
the different CRs are for testing. Should I remove them? In the comment above I wrote what should be expected of each of the CRs

Reply:
Yes remove them if they are for testing.